manta-pay/src/test/compatibility.rs

```diff
@@ -0,0 +1,143 @@
+    manta_parameters::pay::testnet::proving::PrivateTransfer::download(&private_transfer_path)?;
+    let reclaim_path = directory.join("reclaim.dat");
+    manta_parameters::pay::testnet::proving::Reclaim::download(&reclaim_path)?;
+    println!("mint_path: {:?}", mint_path);
+    let proving_context = MultiProvingContext {
+        mint: ProvingContext::decode(IoReader(File::open(mint_path)?))
+            .expect("Unable to decode MINT proving context."),
+        private_transfer: ProvingContext::decode(IoReader(File::open(private_transfer_path)?))
+            .expect("Unable to decode PRIVATE_TRANSFER proving context."),
+        reclaim: ProvingContext::decode(IoReader(File::open(reclaim_path)?))
+            .expect("Unable to decode RECLAIM proving context."),
+    };
+    let verifying_context = MultiVerifyingContext {
+        mint: VerifyingContext::decode(
+            manta_parameters::pay::testnet::verifying::Mint::get()
+                .expect("Checksum did not match."),
+        )
+        .expect("Unable to decode MINT verifying context."),
+        private_transfer: VerifyingContext::decode(
+            manta_parameters::pay::testnet::verifying::PrivateTransfer::get()
+                .expect("Checksum did not match."),
+        )
+        .expect("Unable to decode PRIVATE_TRANSFER verifying context."),
+        reclaim: VerifyingContext::decode(
+            manta_parameters::pay::testnet::verifying::Reclaim::get()
+                .expect("Checksum did not match."),
+        )
+        .expect("Unable to decode RECLAIM verifying context."),
+    };
+    let parameters = Parameters {
+        note_encryption_scheme: NoteEncryptionScheme::decode(
+            manta_parameters::pay::testnet::parameters::NoteEncryptionScheme::get()
+                .expect("Checksum did not match."),
+        )
+        .expect("Unable to decode NOTE_ENCRYPTION_SCHEME parameters."),
+        utxo_commitment: UtxoCommitmentScheme::decode(
+            manta_parameters::pay::testnet::parameters::UtxoCommitmentScheme::get()
+                .expect("Checksum did not match."),
+        )
+        .expect("Unable to decode UTXO_COMMITMENT_SCHEME parameters."),
+        void_number_commitment: VoidNumberCommitmentScheme::decode(
+            manta_parameters::pay::testnet::parameters::VoidNumberCommitmentScheme::get()
+                .expect("Checksum did not match."),
+        )
+        .expect("Unable to decode VOID_NUMBER_COMMITMENT_SCHEME parameters."),
+    };
+    println!("Loading parameters Done.");
+    Ok((
+        proving_context,
+        verifying_context,
+        parameters,
+        UtxoAccumulatorModel::decode(
+            manta_parameters::pay::testnet::parameters::UtxoAccumulatorModel::get()
+                .expect("Checksum did not match."),
+        )
+        .expect("Unable to decode UTXO_ACCUMULATOR_MODEL."),
+    ))
+}
+
+/// Test validity on sampled transactions.
```

Comment:
Update docs: "Test that the circuit has not changed against the current known parameters in `manta-parameters`" (something like this)